addons/mail/static/src/new/core/messaging.js

```diff
@@ -188,6 +189,31 @@ export class Messaging {
             canLeave,
             isAdmin,
         });
+        this.fetchChannelMembers(thread.id);
```

Comment:
I don't like this fetch of channel members whenever a channel is being created.
It encourages rest of code to rely too much on known data as being "complete".
To the extreme, we might just fetch every discuss objects all the time...

Ok to merge this PR for now, because it cleans code overall.
But just know that this fetch is not ok.
We should only fetch when we actually need it.

...And it's not ok to just move the problem to `init_messaging`, as this would definitely introduce huge performance issues at page load (like messaging menu spinner taking ages to load).

Reply:
Then we should at least convert the data when getting thread serverData as what we did before. I can do the follow up pr if you want